backend/entities/PropertyTypes.cs

```diff
@@ -0,0 +1,11 @@
+namespace Pims.Dal.Entities
+{
+    /// <summary>
+    /// PropertyTypes enum, provides the valid property types that can be used.
+    /// </summary>
+    public enum PropertyTypes
```

Comment:
Makes the code cleaner when referring to PropertyTypes, otherwise we have magic integers.